content/storing-in-cloud-must-try.md

```diff
@@ -0,0 +1,69 @@
+[Google Drive + Office Apps](https://www.coolcatteacher.com/how-to-add-google-drive-to-microsoft-word/)
+
+---
+
+##What I feel good about?
+###1. Place-less working
+Ever since I put my important docs such as Thesis, Internship Report and other i-can't-lose document, i can do it from anywhere anytime, because it is sync in the cloud.
+For example, at home i can do my Thesis with my Desktop PC, but when i go out to somewhere i can bring my laptop along and access the work from the cloud or even accessing it from my mobile phone.
+When i store my docs in cloud, i can save it, turn-off my PC and just walk away bringing my laptop because the data is in the cloud.
+If i store it in my PC-local-disk, i should copy-paste it to my laptop first before going out, or maybe store it in a flash drive that i should carry around (more stuffs to be carried).
+
+###2. Safety(?)
+At least, physical safeness is pretty much guaranteed.
+Imagine a scenario, where your house get robbed and Cloud Data Center get robbed.
+In your house, you could lost your laptop or PC, leading to lose of data.
+In Cloud Data Center, even if they get robbed (_which is not likely to happen_), Cloud Data Center guaranteed duplication or backup of your data to fulfill the Service Level Agreement and Standard Operation Procedure and Disaster Prevention.
+This also point back to previous part about flash drive, i might lose my flash drive too, it's small-in-shape.
+Some Cloud Storage provider even provide version rollback and history of editing. If you working on document that involved several people, this feature will be useful to hunt who is making bad changes and praise the one that making good changes.
+[Version history in OneDrive](https://support.microsoft.com/en-us/office/restore-a-previous-version-of-a-file-stored-in-onedrive-159cad6d-d76e-4981-88ef-de6e96c93893)
+
+---
+
+##My Concern
+###1. I need internet
+Because the docs is in the cloud, of course you need internet to synchronize the data between your local disk and cloud. If i am about go out, i need to make sure i have internet-data to access my data in the cloud (_at least, to download it, and edit it locally, and then upload it again_).
+
+###2. Small-small no problem
+Since it require internet connection, i do not recommend to upload big-chunk of data to Cloud storage, like movie or anime collections, because i personally live in an area that do not have high-speed internet speed. That is why, so far i only use it to backup documents, excel file, or power point. Only at certain time i need to share video of work or pictures, that i upload it to cloud storage.
+Internet is not the only problem, price is also the problem. Some cloud storage offer 1-15 GB of storage at free tier. This might not be a problem if you only use cloud storage for storing documents, excel, or power point, but if you use cloud storage as a backup solution, you will need more than 15GB especially if you are storing videos or pictures.
```

Comment:
```suggestion
Since it requires internet connection, I do not recommend uploading a big-chunk of data, like movies or anime collections, to the Cloud Storage because I personally live in an area that doesn't have a high-speed internet speed. That is why, so far I only use it to backup documents, Excel files, or PowerPoint slides. I only upload videos or pictures to the Cloud Storage when I want to share it.
Internet is not the only problem, the price is also the problem. Some Cloud Storage offer 1-15 GB of storage at free tier. This might not be a problem if you only use it to store documents, Excel files, or PowerPoint slides, but if you use Cloud Storage as a backup solution, you will need more than 15 GB, especially if you are storing videos or pictures.
```